Thegaijin/reacipe
# #155024544 Edit category

#### What does this PR do?
Add edit category functionality

#### Description of the task to be completed
The user should be able to create a category in the app

#### How should this be manually tested?
Assuming the user is already logged in, on the dashboard they will be able to see some categories listed. Click on a category name, this opens a collapsible. You will see a category description and some buttons. Click on the edit button. This is bring up a form with the category details. Edit what you would like and submit 

#### What are the relevant pivotal tracker stories?
[#155024544](https://www.pivotaltracker.com/story/show/155024544)
